### ISSUE TITLE
Docker image build fails on ui-crate branch

### ISSUE THREAD

**Issue by StellaV** [NLnetLabs/routinator#646]
Hi all,

I get the following error when I'm trying to build the docker image from the ui-crate branch:

```
#9 54.26 error: failed to run custom build command for `openssl-sys v0.9.65`
#9 54.26 
#9 54.26 Caused by:
#9 54.26   process didn't exit successfully: `/tmp/routinator/target/release/build/openssl-sys-699ed660c5c56321/build-script-main` (exit code: 101)
#9 54.26   --- stdout
#9 54.26   cargo:rustc-cfg=const_fn
#9 54.26   cargo:rerun-if-env-changed=X86_64_ALPINE_LINUX_MUSL_OPENSSL_LIB_DIR
#9 54.26   X86_64_ALPINE_LINUX_MUSL_OPENSSL_LIB_DIR unset
#9 54.26   cargo:rerun-if-env-changed=OPENSSL_LIB_DIR
#9 54.26   OPENSSL_LIB_DIR unset
#9 54.26   cargo:rerun-if-env-changed=X86_64_ALPINE_LINUX_MUSL_OPENSSL_INCLUDE_DIR
#9 54.26   X86_64_ALPINE_LINUX_MUSL_OPENSSL_INCLUDE_DIR unset
#9 54.26   cargo:rerun-if-env-changed=OPENSSL_INCLUDE_DIR
#9 54.26   OPENSSL_INCLUDE_DIR unset
#9 54.26   cargo:rerun-if-env-changed=X86_64_ALPINE_LINUX_MUSL_OPENSSL_DIR
#9 54.26   X86_64_ALPINE_LINUX_MUSL_OPENSSL_DIR unset
#9 54.26   cargo:rerun-if-env-changed=OPENSSL_DIR
#9 54.26   OPENSSL_DIR unset
#9 54.26   cargo:rerun-if-env-changed=OPENSSL_NO_PKG_CONFIG
#9 54.26   cargo:rerun-if-env-changed=PKG_CONFIG
#9 54.26   cargo:rerun-if-env-changed=OPENSSL_STATIC
#9 54.26   cargo:rerun-if-env-changed=OPENSSL_DYNAMIC
#9 54.26   cargo:rerun-if-env-changed=PKG_CONFIG_ALL_STATIC
#9 54.26   cargo:rerun-if-env-changed=PKG_CONFIG_ALL_DYNAMIC
#9 54.26   cargo:rerun-if-env-changed=PKG_CONFIG_PATH_x86_64-alpine-linux-musl
#9 54.26   cargo:rerun-if-env-changed=PKG_CONFIG_PATH_x86_64_alpine_linux_musl
#9 54.26   cargo:rerun-if-env-changed=HOST_PKG_CONFIG_PATH
#9 54.26   cargo:rerun-if-env-changed=PKG_CONFIG_PATH
#9 54.26   cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR_x86_64-alpine-linux-musl
#9 54.26   cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR_x86_64_alpine_linux_musl
#9 54.26   cargo:rerun-if-env-changed=HOST_PKG_CONFIG_LIBDIR
#9 54.26   cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR
#9 54.26   cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR_x86_64-alpine-linux-musl
#9 54.26   cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR_x86_64_alpine_linux_musl
#9 54.26   cargo:rerun-if-env-changed=HOST_PKG_CONFIG_SYSROOT_DIR
#9 54.26   cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR
#9 54.26   run pkg_config fail: "Failed to run `\"pkg-config\" \"--libs\" \"--cflags\" \"openssl\"`: No such file or directory (os error 2)"
#9 54.26 
#9 54.26   --- stderr
```

Could you help me out or am I doing something wrong? :)

Kind regards,
Stella

**Comment by partim** [comment]
Looks like this branch pulls in a dependency on OpenSSL that it shouldn’t. We are looking at it right now. Once that’s fixed, we’ll also merge the branch into main.

**Comment by partim** [comment]
We’ve just released 0.10.1-rc1 with which Docker builds should be working again.

**Comment by StellaV** [comment]
Works great, thanks :)
